library/src/scripts/components/DeviceChecker.tsx

```diff
@@ -60,10 +62,15 @@ export default class DeviceChecker extends React.Component<IDeviceCheckerProps>
     }
 
     /**
-     * @inheritDoc
+     * There's a bug in webpack and there's no way to know the styles have loaded from webpack. In debug mode,
      */
     public componentDidMount() {
         window.addEventListener("resize", this.throttledUpdateOnResize);
+        if (get(window, "gdn.meta.context.debug", false)) {
```

Comment:
~~~You can replace this with `getMeta` from `@library/application`. `getMeta('context.debug')`.~~~

On second thought, since this is a hot reload issue only, we could do a more accurate check. `if (module.hot) {}`  would be more appropriate.